example/otel-collector-config.yml

```diff
@@ -0,0 +1,19 @@
+exporters:
+  debug:
+  otlp:
+    endpoint: jaeger:4317
+    tls:
+      insecure: true
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+service:
+  pipelines:
+    traces:
+      exporters:
+        - debug
```

Comment:
This cause the collector to log. I used it to confirm the proxy to collector connection before I set up Jaeger to view the trace.